package/yast2-network.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon Feb 22 11:52:09 UTC 2016 - mfilka@suse.com
+
+- bnc#955260
+  - Moved setting network service into 1st stage
+- 3.1.140.2
```

Comment:
The SP1 and master versioning has diverged, but you are applying a SP1 version to master.